tests/infra/rates.py

```diff
@@ -62,7 +62,7 @@ def process_next(self):
 
     def get_metrics(self):
         with self.primary.client() as client:
-            rv = client.get("/node/metrics")
+            rv = client.get("/app/metrics")
```

Comment:
Why this change?

Reply:
This was just wrong before, we want metrics for application calls!